### PR TITLE
Refactor issue expiration email

### LINKED ISSUE #2061: Update Issue Expiring Email
### User Story

As a funder I'd like to understand all of my options as it relates to an expired issue.

### Description

[comment]: # (Feature or Bug? i.e Type: Bug)
*Type*: Improvement

### Current Behavior
[comment]: # (Describe what actually happened.)

### Definition of Done
[comment]: # (Any other information that would be useful, bullets are helpful.)

- [x] Design is updated to account for extend expiration and cancel issue
- [ ] Updates are applied to the email template (code)

### Additional Information

screen shot 2018-08-22 at 9 27 07 am
screen shot 2018-08-22 at 9 27 17 am

### PR DESCRIPTION
**Pull request by SaptakS**
##### Description
Refactor UI of issue expiration email to promote extending expiration date.

screen shot 2018-09-05 at 4 53 11 pm

##### Checklist

- [x] linter status: 100% pass
- [x] changes don't break existing behavior
- [x] commit message follows [commit guidelines](https://github.com/gitcoinco/web/blob/master/docs/CONTRIBUTING.md#step-4-commit)

##### Affected core subsystem(s)

email
##### Refers/Fixes

Fixes #2061

### CODE REVIEW

File: app/retail/templates/emails/new_bounty_expire_warning.html [hunk 1]
```diff
@@ -20,32 +20,37 @@
...
 
     <br>
     <br>
-    {% blocktrans %}After the funded issue has expired, you can still submit and accept work.  You may also 'Cancel' the bounty and get the funds back.{% endblocktrans %}
-    <br>
+    {% trans "After a funded issue has expired, a contributor is still able to submit work and you will still be able to accept work." %}
     <br>
-    <h5>{% trans "Look for this button" %}</h5>
     <br>
-    <img src="{% static "v2/images/cancel.png" %}" / >
```
Comment (thelostone-mc) [inline app/retail/templates/emails/new_bounty_expire_warning.html]: Is this used anywhere else ?

**Comment by PixelantDesign** [comment]
looks good!

File: app/retail/templates/emails/new_bounty_expire_warning.html [hunk 2]
```diff
@@ -20,32 +20,37 @@
...
+<p style="color: #4A4A4A;">
     {% if is_owner %}
         {% blocktrans %}Heads up! The funded issue that you posted (below) is expiring in {{num}} {{unit}}.{% endblocktrans %}
     {% else %}
         {% blocktrans %}Heads up! The funded issue that you fulfilled (below) is expiring in {{num}} {{unit}}.{% endblocktrans %}
     {% endif %}
 
+    <br>
+    <br>
+    <img src="{% static "v2/images/expiring_clock.png"%}" style="padding: 20px 0;">
```
Comment (thelostone-mc) [inline app/retail/templates/emails/new_bounty_expire_warning.html]: would you hate me if we used svg here instead ? :P

Comment (SaptakS) [reply app/retail/templates/emails/new_bounty_expire_warning.html]: No I won't. But won't SVG finally get converted to PNG after [this](https://github.com/gitcoinco/web/pull/2106)? @mbeacom should we still be using SVG to maintain proper quality of image?

**Pull request merged**